regression/tests/helpers/api_clients.py

```diff
@@ -140,9 +140,11 @@ class LmsLoginApi(LoginApiBaseClass):
     """
     Login api for LMS
     """
-    def __init__(self):
+    def __init__(self, target_page=None):
```

Comment:
nit: `target_page_name='/dashboard'`, this way you don't need to set this value again in method.